debug_output: true

fit_clothoid: true  # Fit a clothoid before fitting cubic splines?

# Spline settings
minimum_waypoint_distance: 2.0  # Ignore waypoints closer than this distance together
spline_sample_distance: 1.0  # distance between sampled waypoints on the spline
clothoid_point_per_xm: 1

# Visual settings
scale: 1.0